pom.xml

```diff
@@ -602,10 +602,11 @@
                 </requireMavenVersion>
                 <requireProperty>
                   <regexMessage>
-                    x86_64 JDK must be used.
+                    x86_64/aarch64 JDK must be used.
                   </regexMessage>
                   <property>os.detected.arch</property>
                   <regex>^x86_64$</regex>
+                  <regex>^aarch_64$</regex>
```

Comment:
I guess this needs to be:

`<regex>^(x86_64)|(aarch_64)$</regex>

Reply:
or `^(x86_64|aarch_64)$`